libs/application-generic/src/services/analytic-logs/workflow-run/workflow-run.schema.ts

```diff
@@ -54,6 +55,9 @@ const schemaDefinition = {
 
   // Data retention
   expires_at: { type: CHDateTime64(3, 'UTC') },
+
+  severity: { type: CHNullable(CHString()) }, // severity of the workflow run
+  critical: { type: CHNullable(CHBoolean()) }, // critical flag of the workflow run
```

Comment:
we can make those not nullable wit default values, it is more performant in ClickHouse to have any other value than null if possible.

```suggestion
  severity: { type: CHLowCardinality(CHString('none')) }, // severity of the workflow run
  critical: { type:CHBoolean(false) }, // critical flag of the workflow run
```

ALTER TABLE workflow_runs 
ADD COLUMN IF NOT EXISTS severity String DEFAULT 'none';

ALTER TABLE workflow_runs 
ADD COLUMN IF NOT EXISTS critical Boolean DEFAULT false';

Reply:
For the existing `workflow_runs` records, it will populate with the default value, for example, `none` but it might not be the correct value for that record; that's why we need the migration for the old records to fill it with the correct `severity, critical` values